tests/demos/test_notebooks_run.py

```diff
@@ -12,11 +12,11 @@
 @pytest.fixture(params=glob.glob(os.path.join(nb_dir, "*.ipynb")),
                 ids=lambda x: os.path.basename(x))
 def ipynb_file(request):
-    if request.param.endswith("08-composable-solvers.ipynb"):
-        pytest.xfail("FIX THIS LATER: Have to update our PETSc fork to fix build issues")
+    # Notebook 08-composable-solvers.ipynb still has an issue, the cell is commented out
```

Comment:
This comment isn't terribly clear. Reference an issue or provide more explanation?

Reply:
In my defence, nor was the original `xfail` message!

Reply:
https://github.com/firedrakeproject/firedrake/issues/3308